source/modules/MEI.header.xml

```diff
@@ -2550,9 +2550,12 @@
       <rng:zeroOrMore>
         <rng:ref name="model.headLike"/>
       </rng:zeroOrMore>
-      <rng:oneOrMore>
-        <rng:ref name="source"/>
-      </rng:oneOrMore>
+      <choice>
+        <ref name="p"/>
+        <oneOrMore>
+          <ref name="source"/>
+        </oneOrMore>
+      </choice>
```

Comment:
```suggestion
      <rng:choice>
        <rng:ref name="p"/>
        <rng:oneOrMore>
          <rng:ref name="source"/>
        </rng:oneOrMore>
      </rng:choice>
```

Reply:
@musicEnfanthen I can't accept this suggestion, because I do not have writing access here. But you are right. `<choice>` is taken from the rng-namespace, so this has to be added.